packages/grid/src/vaadin-grid-filter.js

```diff
@@ -35,7 +37,7 @@ import { Debouncer } from '@vaadin/component-base/src/debounce.js';
  *
  * @extends HTMLElement
  */
-class GridFilter extends class extends PolymerElement {} {
```

Comment:
This extra class was added in https://github.com/vaadin/vaadin-grid/commit/d4a652cc06ed46a6ee64c20fe63ee877a0ca3c4f to prevent internal Polymer methods from being shown in the API docs page, which typically happens when you have a class that extends `PolymerElement` without using any mixins. As we add a mixin, the workaround can be removed 🙈